templates/batchConversion.tpl

```diff
@@ -0,0 +1,31 @@
+{**
+ * templates/batchConversion.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * markup plugin -- displays the ArticlesGrid.
+ *}
+ <link rel="stylesheet" href="{$pluginCssURL}/batch.css" />
```

Comment:
Stylesheets should be added using `$templateMgr->addStyleSheet`.

Reply:
(as much as possible)

Reply:
In this case, I'm not extending the Form class (cf https://github.com/kaschioudi/ojs3-markup/blob/5aef60f0e607879395d40beeb5918debee274ee4/MarkupPlugin.inc.php#L257). Any suggestion where I should plug the call to `addStyleSheet` and `addJavaScript`?

Reply:
The `addStyleSheet` and `addJavaScript` calls are implemented by `TemplateManager`, not `Form`.

Reply:
Ok.

Reply:
https://github.com/kaschioudi/ojs3-markup/commit/7982580e2c680b22c4d317ef1e642b463b0855cf